package.json

```diff
@@ -1 +1,8 @@
-{ "dependencies": { "@stacks/wallet-sdk": "^6.11.3" } }
\ No newline at end of file
+{
+  "scripts": {
+    "generate-mnemonic": "node -e 'import(\"@stacks/wallet-sdk\").then(({generateSecretKey}) => console.log(generateSecretKey()))'"
```

Comment:
This is an interesting approach, right now each script lives as an individual TS file so would've expected it to match the format.

Bun can execute the TS file just like `ts-node` and install dependencies based on the package.json.

Curious why you used the package.json scripts for the functionality instead?

Reply:
Since the current scripts are all intended for the agent to use I didn't want to mix in one that is made for the developer.  However this could in theory be used by the agent at some point so maybe ti can be?